### ISSUE TITLE
:Vimwiki2HTMLBrowse does not work when vimwiki_markdown is in use

### ISSUE THREAD

**Issue by jsn** [vimwiki/vimwiki#1241]
vimwiki version:
```
Os: LinuxOs: Linux
Vim: 800
Branch: HEAD
Revision: 61c513d
Date: 2022-04-09 20:14:04 -0400

```
Using neovim, markdown syntax, vimwiki_markdown gem. Here's my settings:
`let g:vimwiki_list = [{'path': '~/vimwiki/', 'diary_rel_path': 'diary/', 'path_html': '~/vimwiki/html/', 'syntax': 'markdown', 'ext': '.md', 'custom_wiki2html': 'vimwiki_markdown', 'html_filename_parameterization': 1}]
`

Steps to reproduce: create a wiki page with capital letters in filename (e.g. `Something.md`), open it, type `<Leader>whh` -- nothing is produced. Here's my pull request: #1240 .

**Comment by jsn** [comment]
> nothing is produced

Let me be more specific: internally, an html file is promptly generated, but its name is all lowercased (e.g. `Something.md` -> `something.html`). Then vimwiki uses the wrong filename (`Something.html`) with `system_open_link`, and, since that file does not exist, nothing happens.

**Comment by brennen** [comment]
Fixed with #1240.
